pages/03.installation/01.environment/04.lando/docs.md

```diff
@@ -0,0 +1,188 @@
+  cache: redis
+
+services:
+  appserver:
+    composer:
+      phpunit/phpunit: '*'
+    build_as_root:
+      - apt-get update -y
+      - apt-get install -my wget gnupg
+      - curl -sL https://deb.nodesource.com/setup_12.x | bash - # install node + (UF4.2.0 requires 10.12+) https://github
+      - apt-get install -y nodejs
+      - a2enmod headers
+    #config:
+    #  conf: ./lando-config/php.ini # uncomment this section to enable your custom php.ini file
+    overrides:
+      environment:
+        PHP_IDE_CONFIG: "serverName=userfrosting.lndo.site"
+    ssl: true
+
+
+  # Add a phpmyadmin db frontend
+  pma:
+
+    # Use the latest version of phpmyadmin
+    type: phpmyadmin
+
+    # The databases you want to look at, this will default to a service called
+    # "database"
+    #
+    # You might want to run `lando info` on your app to see what databases you
+    # have available
+    hosts:
+      - database
+
+# Add in a proxy route to phpmyadmin
+proxy:
+  pma:
+    - pma.userfrosting.lndo.site
+
+tooling:
+  phpunit:
+    service: appserver
+    description: "Run PHP Unit tests: lando phpunit"
+  composer:
+
+    # Run the `lando composer` command against the appserver from the services
+    # section above
+    service: appserver
+
+    # Give a nice description to describe what this does.
+    # If ommited this will default to "Run COMMAND commands" where COMMAND is
+    # composer in this case
+    description: Run composer commands
+
+    # Define a specific command to start with inside the container. If omitted
+    # this will default to the name you give the service eg composer.
+    #
+    # This can be a string or an array if you want to run multiple commands
+    # See below for a multi-command example
+#    cmd: composer --ansi
+  npm:
+    service: appserver
+  node:
+    service: appserver
+  gulp:
+    service: appserver
+  redis-cli:
+    service: cache
+```
+
+This configuration file creates a LAMP stack. If you prefer a LEMP (nginx) stack, change "lamp" to "lemp" in the `recipe` setting near the top of the file and be sure to comment out the `a2enmod headers` line in the build section for the Lando config file. This will enable `xdebug` for integration with your IDE. If you do not use xdebug, you can set `xdebug` to false in the configuration file to improve performance. However, since this is a local development environment, we strongly encourage you to keep it enabled!
+
+### Configure UserFrosting
+
+1. Copy `app/sprinkles.example.json` to `app/sprinkles.json`
+1. Run `chmod 777 app/{logs,cache,sessions}` to fix file permissions for web server. (NOTE: File
+   permissions should be properly secured in a production environment!)
+
+### 1st Time Lando Start-up
+
+Using the terminal, run `lando start` from within your UserFrosting app. The first start up will take a brief period to install everything and set up the docker components.
+
+When the application boots successfully, you'll see something like: 
+
+```
```

Comment:
This is wrong. I'd guess a leftover from a previous draft or older lando version maybe. It should be something like;
```
   ___                      __        __        __     __        ______
  / _ )___  ___  __ _  ___ / /  ___ _/ /_____ _/ /__ _/ /_____ _/ / / /
 / _  / _ \/ _ \/  ' \(_-</ _ \/ _ `/  '_/ _ `/ / _ `/  '_/ _ `/_/_/_/ 
/____/\___/\___/_/_/_/___/_//_/\_,_/_/\_\\_,_/_/\_,_/_/\_\\_,_(_|_|_)  
                                                                       

Your app has started up correctly.
Here are some vitals:

 NAME            userfrosting                       
 LOCATION        /your/app/location
 SERVICES        appserver, database, pma           
 APPSERVER URLS  https://localhost:32773            
                 http://localhost:32774             
                 http://userfrosting.lndo.site/     
                 https://userfrosting.lndo.site/    
 PMA URLS        http://localhost:32776             
                 http://pma.userfrosting.lndo.site/ 
```